packages/loader/container-loader/src/container.ts

```diff
@@ -1198,8 +1211,10 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
 
         // ...load in the existing quorum
         // Initialize the protocol handler
+        // this._protocolHandler =
+        //     await this.initializeProtocolStateFromSnapshot(attributes, this.storageService, snapshot);
         this._protocolHandler = pendingLocalState === undefined
```

Comment:
@wes-carlson @ssimic2 -- there were multiple merge conflicts in this file. can you please review this one.